library/spit_results.py

```diff
@@ -356,7 +356,7 @@ def handle_redhat_packages(self, data):
                         max_build_time = pkg.build_time
                         last_built = pkg
 
-                is_red_hat = "Y" if rh_packages > 0 else "N"
+                is_red_hat = "Y" if not rh_packages else "N"
```

Comment:
I think there are two things going on here. The first is that I read `rh_packages > 0` as meaning `len(rh_packages) > 0`. The equivalent if clause is `if rh_packages`, not `if not rh_packages`.

The second is that the Travis build you linked is using Python 3.5. `rh_packages` is constructed using `filter`, and the definition of `filter` changed between Python 2 and 3. In Python 2 `rh_packages` will be a list, but in Python 3 `rh_packages` will be an iterator (https://docs.python.org/3/library/functions.html#filter). The implicit false construct that works for lists will not work for iterators, so we actually can't use it here!

It looks like we use `len(rh_packages)` a few times below that point in the file, and those will raise TypeError in Python 3. I think your two options are to either convert `rh_packages` to a list using `rh_packages = list(rh_packages)`, or to compute its length in the `for pkg in rh_packages` loop above and then use that everywhere that uses the length now.

I would probably just make it a list, but up to you.